artemis-server/src/test/java/org/apache/activemq/artemis/core/server/protocol/websocket/WebSocketFrameEncoderTest.java

```diff
@@ -78,7 +78,7 @@ public void testWriteNonByteBuf() throws Exception {
 
    @Test
    public void testWriteReleaseBuffer() throws Exception {
-      String content = "Using direct buffer";
```

Comment:
git commit --amend
git push -f

(without changing anything would do to force a build) :)

Reply:
thanks for the tip :)